### DISCUSSION TITLE
Verify our identity on Mastodon using `rel=me` attribute

### DISCUSSION THREAD

**Discussion by erinb223** [forem/forem#18822] [Feature Requests]
Hey team! Got a Mastodon-related quick win for ya today: profile verification.

The most effective way to verify our identity on Mastodon is to add a link with a `rel=me` attribute to both our site homepage (dev.to) and our GitHub repo (github.com/forem/forem). [Mastodon has documentation on how to do it and how it works](https://docs.joinmastodon.org/user/profile/#verification).

This is what our profile looks like currently:
![A screenshot of our Mastodon profile, with all our links boxed in standard gray.](https://user-images.githubusercontent.com/66445787/206271752-c33ab8c0-c613-4f47-b915-22f5b672c440.png)

Once we have verified our ownership of our domain and our repo using the above method, they will look like these:
![A screenshot of Scott Hanselman's Mastodon profile, which shows three of his links boxed in green with a checkmark.](https://user-images.githubusercontent.com/66445787/206272056-03dd0523-71c3-45b6-be9c-c33179345b93.png)

One way to incorporate this on the dev.to homepage, while also encouraging Mastodon users to engage with us on their social media platform of choice, could be to add a Mastodon icon to our social media icon selection here:
![A screenshot of our homepage with the social media icons circled in red](https://user-images.githubusercontent.com/66445787/206273141-953f248a-eb99-4609-824f-02633f285700.png)

Verification on Mastodon is key for the following reasons:
- It proves that we are the real dev.to, not a mirror
- It makes our audience more likely to trust us on the platform
- Users can safely, confidently follow verified links off the platform and onto our sites

Thanks for your consideration! I think this could be something of a quick win and I appreciate your work on it.

**Comment by erinb223** [comment]
Updating to add that our Mastodon profile is now on [Fosstodon](https://fosstodon.org/@thepracticaldev) rather than mastodon.social.

**Comment by unixorn** [comment]
Sorry - dev.to points people here so I assumed they owned the repo which was dumb of me.
